doc/source/guide/introduction/getting_started.rst

```diff
@@ -2,4 +2,288 @@
+and show how to use the respective operators, solvers and spaces in a correct manner.
+
+Example: Solving an inverse problem
+===================================
+In what follows, we will give an example of the workflow one might have when solving an inverse problem as
+it is encountered "in real life". The problem we want to solve is
+
+.. math::
+   
+   Af = g
+
+Where :math:`A` is the `convolution <https://en.wikipedia.org/wiki/Convolution>`_ operator
+
+.. math::
+   
+   (Af)(x) = \int f(x) k(x-y) dy
+
+where :math:`k` is the convolution kernel, :math:`f` is the solution and :math:`g` is known data. As is typical in applications, the convolution operator may not be available in ODL (we'll pretend it's not), so we will need to implement it.
+
+We start by finding a nice implementation of the convolution operator, `scipy happens to have one <http://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.fftconvolve.html>`_, and create a wrapping `Operator` for it in ODL.
+
+.. code-block:: python
+
+   import odl
+   import scipy
+   
+   class Convolution(odl.Operator):
+       """Operator calculating the convolution of a kernel with a vector.
+   
+       The operator inherits from ``odl.Operator`` to be able to be used with odl.
+       """
+   
+       def __init__(self, kernel):
+           """Initialize a convolution operator with a known kernel."""
+   
+           # Store the kernel
+           self.kernel = kernel
+           
+           # Initialize the Operator class by calling its __init__ method.
+           # This sets properties such as domain and range and allows the other
+           # operator convenience functions to work.
+           odl.Operator.__init__(self, domain=kernel.space, range=kernel.space,
+                                 linear=True)
+   
+       def _call(self, x):
+           """Implement calling the operator by calling scipy."""
+           return scipy.signal.convolve(self.kernel, x, mode='same')
+   
+We can verify that our operator works by calling it on some data. This can either come from some outside source, or from simulations. ODL also provides a nice range of standard phantoms such as the `cuboid` and `shepp_logan` phantoms:
+
+.. code-block:: python
+
+   space = odl.uniform_discr([-1, -1], [1, 1], [100, 100])
+   kernel = odl.phantom.cuboid(space, [-0.05, -0.05], [0.05, 0.05])
+   conv = Convolution(kernel)
+   phantom = odl.phantom.shepp_logan(space, modified=True)
+   data = conv(phantom)
+   
+   kernel.show('kernel')
+   phantom.show('phantom')
+   data.show('convolved phantom')
+
+.. image:: figures/getting_started_kernel.png
+
+.. image:: figures/getting_started_phantom.png
+
+.. image:: figures/getting_started_convolved.png
+
+We can use this data to try to solve the inverse problem. To solve that, we need further properties of the operator. Specifically, we need an `Operator.adjoint`. Luckily, the convolution operator is self adjoint if the kernel is symmetric, so we can add: 
```

Comment:
The data is part of the inverse problem, so this sentence is a bit weird. I'd write something along the lines of "We can use this data as right-hand side in our inverse problem."

"To solve that" -> "To solve it" or "To be able to solve it"